### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by Ceemos96**
### In this milestone, I : 
- Made sure that state is immutable.
- Connected the app to redux store.
- Added functionality for the Add new button.
- Added functionality for the removw button.
- Ensured that there are no linter errors

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 import App from './App';
+import store from './redux/ConfigureStore.js';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <BrowserRouter>
-      <App />
+      <Provider store={store}>
+        <App />
```
Comment (Ismailco) [inline src/index.js:15]: - [ ] The app compiles with errors, that prevent it from running, kindly fix this so your app works as expected, you can checkout the naming of the redux folder, it's Redux not redux

Screenshot_20220723_181630

File: src/components/Book.js [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+  return (
+    <div className="book-item">
+      <div className="book-info">
+        <p className="title">{ title }</p>
+        <p className="author">{ author }</p>
+      </div>
+      <ul className="book-actions">
+        <li>Comments</li>
+        <li>
+          <button className="remove" type="button" onClick={handleDelete}>Remove</button>
```
Comment (Ismailco) [inline src/components/Book.js:22]: - [ ] Adding and removing a book is not working, kindly also consider fixing that so the form can add a book, and the remove button removes a book from the list

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Ceemos96, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Ceemos96 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ceemos96** [comment]
Thanks and God bless.

**Comment by vic778** [comment]
Hi @Ceemos96 
# This PR will be invalidated because it has been approved so i can't review it.
Kindly change the PR and submit it in your Dashboard

**Pull request merged**